server/apps/distributed-app/src/main/java/org/apache/james/CassandraRabbitMQJamesServerMain.java

```diff
@@ -205,6 +206,21 @@ public static GuiceJamesServer createServer(CassandraRabbitMQJamesConfiguration
             .combineWith(chooseJmapModule(configuration));
     }
 
+    private static Module chooseMailQueue(CassandraRabbitMQJamesConfiguration configuration) {
+        switch (configuration.getMailQueueChoice()) {
+            case PULSAR:
+                return new PulsarQueueModule();
+            case RABBITMQ:
+                return Modules.combine(
+                    new RabbitMQModule(),
+                    new RabbitMailQueueRoutesModule(),
+                    new RabbitMQMailQueueModule(),
+                    MailQueueViewChoice.ModuleChooser.choose(configuration.getMailQueueViewChoice()));
```

Comment:
Well `RabbitMQModule` should not be in here actually @Arsnael 
Please check again @thanhbv200585

Reply:
Ok, I will revert it